docs/Guide/application-commands/interaction-handlers/modals.mdx

```diff
@@ -3,4 +3,34 @@ sidebar_position: 5
 title: Modals
 ---
 
-<!-- TODO: https://vladfrangu.notion.site/Handling-Interactions-not-Application-Commands-in-Sapphire-92e307005b9f4e3b849663ea43c57701 -->
+Modals are like your annoying pop-ups, but cooler! These interactions will display a pop-up window on the user's client, which when submitted, will fire this event. The great thing about modals is that they're going to be handled just like any other interaction!
+
+:::warning
+
+There is no way to determine if or how the user closed the modal. 
```

Comment:
```suggestion
Currently, there is no way to determine if or how the user closed the modal.  
```

Reply:
This will not be added to the client (at least what I understood from Mason)

Reply:
Sadge but aight then